code/game/gamemodes/changeling/changeling.dm

```diff
@@ -421,12 +423,12 @@ var/list/slot2type = list("head" = /obj/item/clothing/head/changeling, "wear_mas
 	user.underwear = chosen_prof.underwear
 	user.undershirt = chosen_prof.undershirt
 	user.socks = chosen_prof.socks
-	user.dna = chosen_dna //Should fix the whole "ERM MEH GERD I TURNED INTO A HUMAN INSTEAD OF ROBIT"
```

Comment:
We've got an issue here Crystal, that was there to deal with changlings not getting the correct species, unless you managed to handle it elsewhere now.